data/data/bootstrap/systemd/units/openshift.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Bootstrap an Openshift cluster
+Wants=bootkube.service
+After=bootkube.service
+ConditionPathExists=!/opt/openshift/.openshift.done
+
+[Service]
+WorkingDirectory=/opt/openshift/openshift
+ExecStart=/usr/local/bin/openshift.sh /opt/openshift/auth/kubeconfig
```

Comment:
I'd prefer a more specific name for this script and service, since "OpenShift" feels like "Kubernetes plus the OpenShift-specific stuff" and not "just the OpenShift-specific stuff" to me.  The script is about pushing our manifests into the cluster, so maybe `manifests.sh` (and `manifests.service`)?  Or `initial-manifests.sh`?  `operator-manifests.sh`? Or something?  `special-sauce-beyond-vanilla-kubernetes.sh`? ;)

Reply:
I had changed it to `manifests.service` and put the manifests that it used at `/opt/openshift/manifests`, but there were conflicts with manifests coming from another location. I was not in love with the name `manifests.service` as there are a lot of different manifests used for different purposes. If we really care about changing this name, then we can have a discussion around better names and change it in another PR.